fhir-path/src/main/java/com/ibm/fhir/path/util/FHIRPathUtil.java

```diff
@@ -905,6 +905,11 @@ public static String unescape(String s) {
 
         FHIRPathTree tree = evaluator.getEvaluationContext().getTree();
         FHIRPathNode parentNode = tree.getParent(node);
+
+        if (parentNode == null) {
+            throw new FHIRPatchException("Unable to compute the parent for '" + elementName + "';" +
+                    " a FHIRPathPatch replace FHIRPath must select a node with a parent", fhirPath);
+        }
```

Comment:
this is a somewhat unrelated change that might merit its own separate issue, but i found this NPE bug when I added a patch test and got the patch wrong